scripts/purge_and_reinstall.sh

```
@@ -0,0 +1,333 @@
+        echo "❌ Failed to create virtual environment"
+        exit 1
+    fi
+    
+    echo "✅ Virtual environment created"
+}
+
+# fix_symbolic_links removes existing Python and pip symbolic links in the virtual environment and recreates them to point directly to system Python and pip executables.
+fix_symbolic_links() {
+    echo "🔗 Fixing symbolic links..."
+    
+    cd venv/bin
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)